metal-k8s.yml

```diff
@@ -57,14 +57,10 @@
   gather_facts: False
   roles:
     - role: info_displayer
-      add_info:
-        - master_dashboard_url: >-
-            https://{{ kube_login }}:{{ kube_password }}@{{ ip_master }}:6443/ui
-          kube_login: '{{ kube_login }}'
-          kube_password: '{{ kube_password }}'
       vars:
-        kube_login: 'kube'
-        kube_password: >-
-          {{ lookup('file', 'kubespray/credentials/kube_user') }}
-        ip_master: >-
-          {{ hostvars[groups['kube-master'][0]].ansible_default_ipv4.address }}
+        add_info:
+        - master_dashboard_url: >-
+            https://{{ hostvars[groups['kube-master'][0]].ansible_default_ipv4.address
+            }}:6443/api/v1/namespaces/kube-system/services/https:kubernetes-dashboard:/proxy
+          kube_login: 'kube'
```

Comment:
I guess what we really want is to retrieve this out of `kube_users`, but well ;)

Reply:
(was not aware of kube_users). kube_users seems to be a dict containing all user, so how to extract the wanted user ? 
{{ kube_users.popitem() }} (a user at random ?) or something more complex like searching for the first user to be admin ? (but if multiple admin exist , as dict iteration is not consistent, this might be different each time)

Reply:
No need to change now as far as I'm concerned.

We should, at some point, get rid of 'Basic auth' anyway :)

Reply:
Ohhh, agreed on that